modules/images-other-jenkins-agent-images.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * images/using_images/images-other-jenkins-agent.adoc
+
+[id="images-other-jenkins-agent-images_{context}"]
+= Jenkins agent images
+
+The {product-title} Jenkins agent images come in two varieties, Red Hat
+Enterprise Linux (RHEL) 7 based images and CentOS 7 Based Images.
+
+[discrete]
+=== RHEL 7 Based Images
+
+RHEL 7 Jenkins images are available through the Red Hat Registry:
+
+----
+$ docker pull regsitry.redhat.io/openshift4/ose-jenkins:<v4.1.4>
+$ docker pull regsitry.redhat.io/openshift4/ose-jenkins-agent-nodejs:<v4.1.4>
+$ docker pull regsitry.redhat.io/openshift4/ose-jenkins-agent-maven:<v4.1.4>
+$ docker pull regsitry.redhat.io/openshift4/ose-jenkins-agent-base:<v4.1.4>
+----
+
+[NOTE]
+====
+The RHEL 7 Jenkins images exist as an imagestream in {product-title}.
+====
+
+[discrete]
+=== CentOS 7 Based Images
```

Comment:
The CentOS vs. RHEL distinction does not pertain for 4.x

We are based on the UBI, and what you exactly get when the image is brought depends on whether you are on a node with subscriptions or not.

@bmcelvee - are there UBI references elsewhere in the doc you can reference here?

If it helps, I have 3.x and 4.x breakdowns in the README in the openshift/jenkins repo.  Here is the 4.x descriptions:  https://github.com/openshift/jenkins#installation-openshift-v4